x/ethbridge/client/cli/query.go

```diff
@@ -14,19 +16,34 @@ import (
 // GetCmdGetEthBridgeProphecy queries information about a specific prophecy
 func GetCmdGetEthBridgeProphecy(queryRoute string, cdc *codec.Codec) *cobra.Command {
 	return &cobra.Command{
-		Use:   "prophecy [nonce] [ethereum-sender]",
+		Use:   "prophecy [chain-id] [bridge-contract] [nonce] [symbol] [token-contract] [ethereum-sender]",
 		Short: "Query prophecy",
-		Args:  cobra.ExactArgs(2),
+		Args:  cobra.ExactArgs(6),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cliCtx := context.NewCLIContext().WithCodec(cdc)
 
-			nonce, err := strconv.Atoi(args[0])
+			chainID, err := strconv.Atoi(args[0])
```

Comment:
i'm not sure why the chain ID was an int in the first place tbh

Reply:
see comment above